src/pages/[platform]/build-a-backend/auth/add-social-provider/index.mdx

```diff
@@ -559,18 +662,23 @@ function App() {
 <Block name="TypeScript">
 
 ```ts
-import { useEffect, useState } from "react";
-import { Text, View, Linking, Button } from "react-native";
+import React, { useEffect, useState } from "react";
+import {
+  Button,
+  SafeAreaView,
+  Text,
+} from "react-native";
+
 import { Amplify } from "aws-amplify";
-import { Hub } from "aws-amplify/utils";
-import { signInWithRedirect, signOut, getCurrentUser } from "aws-amplify/auth";
-import { AuthUser } from "aws-amplify/auth";
-import config from './amplifyconfiguration.json';
+import { AuthUser, getCurrentUser, signInWithRedirect, signOut } from "aws-amplify/auth";
+import { Hub } from "@aws-amplify/core";
 
-Amplify.configure(config);
+import amplifyconfig from "./amplifyconfiguration.json";
 
-function App() {
-  const [user, setUser] = useState<CognitoAuthUser | null>(null);
+Amplify.configure(amplifyconfig);
+
+function App(): JSX.Element {
```

Comment:
`: JSX.Element`not needed I think

Reply:
I got that when using `react-native` CLI